### PR TITLE
Feat usage daemon

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?
- Create new stats collection per project
- Introduces a Usage daemon that syncs the data from InfluxDB to Appwrite's stats collection for each project
- collects more stats (all documented in https://github.com/appwrite/appwrite/blob/08bdf089f62f84464fc188701683a04c872a7fda/app/tasks/usage.php#L18)
- Update appwrite usage endpoints to use stats collection instead of influxdb
- remove influxdb dependency on appwrite container

## Open Question
- Should we count the number of operations from DB workers (for example, when a collection is deleted, should we count the document delete operations that happen for that collection as well?

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

(If this PR is related to any other PR or resolves any issue or related to any issue link all related PR and issues here.)

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

YES

### CODE REVIEW

**Review by TorstenDittmann — APPROVED** [review]
lgtm 👍🏻

File: app/tasks/usage.php [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+    ->action(function () use ($register) {
+        Console::title('Usage Aggregation V1');
+        Console::success(APP_NAME . ' usage aggregation process v1 has started');
+
+        $interval = (int) App::getEnv('_APP_USAGE_AGGREGATION_INTERVAL', '30'); //30 seconds
+        $periods = [
+            [
+                'key' => '30m',
+                'startTime' => '-24 hours',
+            ],
```
Comment (christyjacob4) [inline app/tasks/usage.php:72]: Are we missing the 30 days period here ? @lohanidamodar

Comment (lohanidamodar) [reply app/tasks/usage.php:72]: Nope, for 30 days we get `1d` period

Comment (christyjacob4) [reply app/tasks/usage.php:72]: Alright :+1:

File: app/controllers/api/account.php [hunk 2]
```diff
@@ -147,13 +152,15 @@
...
-    ->action(function ($email, $password, $request, $response, $dbForInternal, $locale, $geodb, $audits) {
+    ->inject('usage')
+    ->action(function ($email, $password, $request, $response, $dbForInternal, $locale, $geodb, $audits, $usage) {
         /** @var Utopia\Swoole\Request $request */
         /** @var Appwrite\Utopia\Response $response */
         /** @var Utopia\Database\Database $dbForInternal */
         /** @var Utopia\Locale\Locale $locale */
         /** @var MaxMind\Db\Reader $geodb */
         /** @var Appwrite\Event\Event $audits */
+        /** @var Appwrite\Usage\Usage $usage */
```
Comment (christyjacob4) [inline app/controllers/api/account.php:163]:
```suggestion
        /** @var Appwrite\Stats\Stats $usage */
```

File: app/controllers/api/account.php [hunk 3]
```diff
@@ -52,12 +52,14 @@
...
     ->inject('audits')
-    ->action(function ($userId, $email, $password, $name, $request, $response, $project, $dbForInternal, $audits) {
+    ->inject('usage')
+    ->action(function ($userId, $email, $password, $name, $request, $response, $project, $dbForInternal, $audits, $usage) {
         /** @var Utopia\Swoole\Request $request */
         /** @var Appwrite\Utopia\Response $response */
         /** @var Utopia\Database\Document $project */
         /** @var Utopia\Database\Database $dbForInternal */
         /** @var Appwrite\Event\Event $audits */
+        /** @var Appwrite\Usage\Usage $usage */
```
Comment (christyjacob4) [inline app/controllers/api/account.php:62]:
```suggestion
        /** @var Appwrite\Stats\Stats $usage */
```

**Review by eldadfux — APPROVED** [review]
Just waiting for tests to pass, and we can merge.

**Comment by lohanidamodar** [comment]
> Just waiting for tests to pass, and we can merge.

Tests are failing on ARM only and all the errors are Timeout errors, I'm restarting again one more time.

**Comment by lohanidamodar** [comment]
@eldadfux Tests are passing!

**Pull request merged**